src/kaleidoscope/hardware/keyboardio/Model01.cpp

```diff
@@ -16,6 +16,7 @@
  */
 
 #ifdef ARDUINO_AVR_MODEL01
+#ifndef ARDUINO_VIRTUAL
```

Comment:
Make sure that non-virtual keyboard hardwares are not compiled in virtual builds.